python/cuspatial/cuspatial/tests/test_contains.py

```diff
@@ -0,0 +1,362 @@
+import geopandas as gpd
+import pytest
+from shapely.geometry import LineString, Point, Polygon
+
+import cuspatial
+
+
+@pytest.mark.xfail(reason="The polygons share numerous boundaries.")
+def test_manual_polygons():
+    gpdlhs = gpd.GeoSeries([Polygon(((-8, -8), (-8, 8), (8, 8), (8, -8))) * 6])
+    gpdrhs = gpd.GeoSeries(
+        [
+            Polygon(((-8, -8), (-8, 8), (8, 8), (8, -8))),
+            Polygon(((-2, -2), (-2, 2), (2, 2), (2, -2))),
+            Polygon(((-10, -2), (-10, 2), (-6, 2), (-6, -2))),
+            Polygon(((-2, 8), (-2, 12), (2, 12), (2, 8))),
+            Polygon(((6, 0), (8, 2), (10, 0), (8, -2))),
+            Polygon(((-2, -8), (-2, -4), (2, -4), (2, -8))),
+        ]
+    )
+    rhs = cuspatial.from_geopandas(gpdrhs)
+    lhs = cuspatial.from_geopandas(gpdlhs)
+    got = lhs.contains_properly(rhs).values_host
+    expected = gpdlhs.contains(gpdrhs).values
+    assert (got == expected).all()
+    got = rhs.contains_properly(lhs).values_host
+    expected = gpdrhs.contains(gpdlhs).values
```

Comment:
Hmmm, shouldn't you be using `shapely.contains_properly()` for the expected result as we discussed in the meeting?

Reply:
The interfaces are not the same. Using `shapely.contains_properly(x, y)` is a method that takes two Shapely geometries and returns True or False. `.contains` is a `GeoSeries` method that operates on `self` and `other`. Refactoring these tests to use `shapely` only is not comparing apples to oranges.

Reply:
Naively you can use a for-loop...
```
expected = pd.Series()
for lhs, rhs in zip(gpdlhs, gpdrhs):
    expected = pd.concat([expected, [shapely.contains_properly(lhs, rhs)])
```

Reply:
But comparing cuspatial.contains_properly` to `geopandas.contains` *is* comparing apples to oranges.